Expensify/App
# refactor CreateMenu to have dynamic items

### Details
- updated `CreateMenu` to take a dynamic list of items based on `MENU_ITEM_KEYS` enum
- updated `ReportActionCompose` to open the original attachment modal when `AttachmentPicker` is selected

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify.cash/issues/1508

### Tests


1. log in and click on the + fab to verify list has new items
2. enter a convo with more than one person to see `SplitBill|AddAttachment` options when clicking the attach icon
3. enter a convo with one person to see `RequestMoney |AddAttachment` options when clicking the attach icon


### Tested On


### Video Walkthrough

https://user-images.githubusercontent.com/12075600/108810552-3ca9c000-7560-11eb-9ed2-ce4325973ec9.mov


### Screenshots


#### Web

<img width="1676" alt="Screen Shot 2021-02-22 at 10 22 47 PM" src="https://user-images.githubusercontent.com/12075600/108810415-f0f71680-755f-11eb-9ab1-55c0ff79a9d0.png">

#### Mobile Web

<img width="437" alt="Screen Shot 2021-02-22 at 10 50 49 PM" src="https://user-images.githubusercontent.com/12075600/108810657-7b3f7a80-7560-11eb-9210-b9117cf248c1.png">

#### Desktop

<img width="1146" alt="Screen Shot 2021-02-22 at 10 12 33 PM" src="https://user-images.githubusercontent.com/12075600/108810500-1ab03d80-7560-11eb-9d91-8f8fd69ca965.png">

#### iOS

<img width="401" alt="Screen Shot 2021-02-22 at 10 07 21 PM" src="https://user-images.githubusercontent.com/12075600/108810513-2439a580-7560-11eb-8332-2beb61504f35.png">

#### Android

<img width="501" alt="Screen Shot 2021-02-22 at 10 29 25 PM" src="https://user-images.githubusercontent.com/12075600/108810526-2e5ba400-7560-11eb-990e-5e00722e54ae.png">
